.pep8speaks.yml

```diff
@@ -1,4 +1,4 @@
 pycodestyle:
+   max-line-length: 120
    ignore:  # Errors and warnings to ignore
-     - E501  # Line too long
-
+     - w391 # trailing empty lines. More a test than anything else, can be removed.
```

Comment:
It looks like that `pycodestyle` is case-sensitive. In that case I should transform the errors to uppercase.
```
➜  pep8speaks git:(master) pycodestyle handlers.py 
handlers.py:79:80: E501 line too long (84 > 79 characters)
handlers.py:104:5: E266 too many leading '#' for block comment
handlers.py:106:5: E266 too many leading '#' for block comment
handlers.py:109:5: E266 too many leading '#' for block comment
handlers.py:110:5: E266 too many leading '#' for block comment
handlers.py:111:5: E266 too many leading '#' for block comment
handlers.py:123:80: E501 line too long (92 > 79 characters)
handlers.py:200:1: E303 too many blank lines (3)
➜  pep8speaks git:(master) pycodestyle handlers.py --ignore=e501
handlers.py:79:80: E501 line too long (84 > 79 characters)
handlers.py:104:5: E266 too many leading '#' for block comment
handlers.py:106:5: E266 too many leading '#' for block comment
handlers.py:109:5: E266 too many leading '#' for block comment
handlers.py:110:5: E266 too many leading '#' for block comment
handlers.py:111:5: E266 too many leading '#' for block comment
handlers.py:123:80: E501 line too long (92 > 79 characters)
handlers.py:200:1: E303 too many blank lines (3)
```